src/Context/TaskContext.jsx

```diff
@@ -0,0 +1,58 @@
+import { createContext, useEffect, useState } from "react";
+import { v4 as uuidv4 } from "uuid";
+const TaskContext = createContext();
```

Comment:
You don't really have to use `contextAPI` here. You are already using `localStorage` to store your tasks. Adding `contextAPI` on top of that doesn't have any benefit. Instead, it adds an extra layer of complexity. But I'm glad you implemented `contextAPI` here. Good Job. 👍🏼